.travis.yml

```diff
@@ -25,7 +24,7 @@ deploy:
     secure: keSFPLzVj/yAAJ8kw4m+U5kpIiGqubpDqhcHPlGAvMohFMIO4D8ffgCqJdbhQshTtb/wiS5ajK1KNpEzq4BKD/5oRXuj/sIRTMENzX/azUXkT/bADLEGocyfE4/NAGQ9mcDOmHntjt7UUhYN6H87rgex22er4V1vD5tGf5Kc1HU=
   on:
     tags: true
-    repo: GoogleCloudPlatform/gcloud-python
+    repo: GoogleCloudPlatform/google-cloud-python
     # until this is fixed: https://github.com/travis-ci/travis-ci/issues/1675
```

Comment:
Is this going to break when we merge?  I'm guessing we will be renaming the repository soonish afterwards.